scripts/mac/agent.bash

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#################################################################################################################################
+####################################################### Agent Mac Script ########################################################
+#################################################################################################################################
+
+# Gets agent functions scripts from logzio-agent-manifest repo to logzio-temp directory
+# Error:
+#   Exit Code 1
+function get_agent_functions_scripts () {
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Getting agent functions script file from logzio-agent-manifest repo ..." >> $log_file
+    curl -fsSL $repo_url/scripts/mac/functions.bash > $logzio_temp_dir/agent_functions.bash 2>$task_error_file
+    if [[ $? -ne 0 ]]; then
+        local err=$(cat $task_error_file)
+        echo -e "[ERROR] [$(date +"%Y-%m-%d %H:%M:%S")] agent.bash (1): failed to get agnet functions script file from logzio-agent-manifest repo.\n  $err" >> $log_file
+        echo -e "\033[0;31magent.bash (1): failed to get agnet functions script file from logzio-agent-manifest repo.\n  $err\033[0;37m"
+        rm -R logzio-temp
+        exit 1
+    fi
+
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Getting utils functions script file from logzio-agent-manifest repo ..." >> $log_file
+    curl -fsSL $repo_url/scripts/mac/utils_functions.bash > $logzio_temp_dir/utils_functions.bash 2>$task_error_file
+    if [[ $? -ne 0 ]]; then
+        local err=$(cat $task_error_file)
+        echo -e "[ERROR] [$(date +"%Y-%m-%d %H:%M:%S")] agent.bash (1): failed to get utils functions script file from logzio-agent-manifest repo.\n  $err" >> $log_file
+        echo -e "\033[0;31magent.bash (1): failed to get utils functions script file from logzio-agent-manifest repo.\n  $err\033[0;37m"
+        rm -R logzio-temp
+        exit 1
+    fi
+}
+
+
+# Consts
+repo_url="https://raw.githubusercontent.com/logzio/logzio-agent-manifest/v0.2"      # logzio-agent-manifest repo URL
```

Comment:
should be latest and not v0.2

Reply:
I use this to get the files in the repo, it should be master not latest. I will change it after the tests.